models/data/IPortfolio.ts

```diff
@@ -1,4 +1,36 @@
 /** Portfolio data */
+export interface IWorkHistory {
+  position?: string;
+  company?: string;
+  dateWorked?: string;
+  description?: string;
+}
+export interface IProjects {
+  title?: string;
+  picture?: string;
+  summary?: string;
+  viewCode?: string;
+  liveDemo?: string;
+}
+
+export interface ISkills {
```

Comment:
This one can stay as `ISkills` since it contains two types of skills (tech and soft)